Source/Renderer/Context.js

```diff
@@ -269,11 +269,11 @@ define([
         this._antialias = gl.getContextAttributes().antialias;
 
         // Query and initialize extensions
-        this._standardDerivatives = !!getExtension(gl, ['OES_standard_derivatives']);
-        this._elementIndexUint = !!getExtension(gl, ['OES_element_index_uint']);
-        this._depthTexture = !!getExtension(gl, ['WEBGL_depth_texture', 'WEBKIT_WEBGL_depth_texture']);
-        this._textureFloat = !!getExtension(gl, ['OES_texture_float']);
-        this._fragDepth = !!getExtension(gl, ['EXT_frag_depth']);
+        this._standardDerivatives = !!getExtension(gl, ['OES_standard_derivatives']) || this._webgl2;
+        this._elementIndexUint = !!getExtension(gl, ['OES_element_index_uint']) || this._webgl2;
+        this._depthTexture = !!getExtension(gl, ['WEBGL_depth_texture', 'WEBKIT_WEBGL_depth_texture']) || this._webgl2;
+        this._textureFloat = !!getExtension(gl, ['OES_texture_float']) || this._webgl2;
```

Comment:
This extension isn't supported in WebGL 2. To render to a floating point texture, you need support for the EXT_color_buffer_float extension.